client/landing/gutenboarding/style.scss

```diff
@@ -58,6 +58,10 @@ $admin-sidebar-width-collapsed: 0;
 					}
 				}
 			}
+
+			&.is-selected::before {
+				display: none;
+			}
```

Comment:
@yansern Can you take a look at this? I added it without much investigation to remove this selected block UI:

![Screen Shot 2020-02-10 at 17 58 09](https://user-images.githubusercontent.com/841763/74172568-c413e200-4c30-11ea-81b9-27149551526b.png)

I believe you've been looking at this area recently.

Reply:
This is fine. Awaiting the next renovate PR to update `@wordpress/block-editor` to `3.7.0` (as seen here https://github.com/WordPress/gutenberg/commit/b3f900c34d41c282db5a6d5a43f8eaf5f091f3d6) then we can finally get rid of all the CSS.

Related: https://github.com/Automattic/wp-calypso/issues/38635